tests/components/plugwise/test_sensor.py

```diff
@@ -111,16 +111,7 @@ async def test_p1_3ph_dsmr_sensor_entities(
     assert float(state.state) == 2080.0
 
     state = hass.states.get("sensor.p1_voltage_phase_one")
-    assert state
-    assert float(state.state) == 233.2
-
-    state = hass.states.get("sensor.p1_voltage_phase_two")
-    assert state
-    assert float(state.state) == 234.4
-
-    state = hass.states.get("sensor.p1_voltage_phase_three")
-    assert state
-    assert float(state.state) == 234.7
+    assert not state
```

Comment:
It would be good to enable the sensor and check that it is working, example how to enable:
https://github.com/home-assistant/core/blob/2511402400ff711154f2b721339c5b01c1ffe998/tests/components/shelly/test_update.py#L67-L73

Reply:
Reinstated as suggested